index.d.ts

```diff
@@ -3001,11 +3001,11 @@ declare module sequelize {
   }
 
   export interface OrOperator {
-    $or: WhereOperators | WhereAttributeHash | Array<WhereOperators | WhereAttributeHash>;
+    $or: WhereOperators | WhereAttributeHash | Array<Array<number> | WhereOperators | WhereAttributeHash>;
```

Comment:
@felixfbecker I just tried it and looks like Sequelize treats an array as an `$in` automatically and generates something like this:
```sql
SELECT [id], [created_at], [updated_at] FROM [User] AS [User] WHERE ([User].[id] IN (1, 2, 3) OR [User].[id] > 10
```